modules/client/test/integration/client/methods.test.ts

```diff
@@ -107,7 +114,15 @@ describe("Client", () => {
           },
           ensSubdomain: daoName.toLowerCase().replace(" ", "-"),
           plugins: [
-            { id: "0x1234", data: new Uint8Array([11, 11]) },
+            {
+              id: deployment.allowListRepo.address,
+              data: toUtf8Bytes(
+                ethers.utils.defaultAbiCoder.encode(
```

Comment:
Could we explicitly import the utils.defaultAbiCoder, so that we don't need `ethers` at all?